plugins/jobtype/src/azkaban/jobtype/HadoopConfigurationInjector.java

```diff
@@ -50,6 +52,17 @@
    * configuration properties to automatically inject.
    */
   public static void injectResources() {
+    // Add mapred, yarn and hdfs site configs (in addition to core-site, which
+    // is automatically added) as default resources before we add the injected
+    // configuration. This will cause the injected properties to override the
+    // default site properties (instead of vice-versa). This is safe to do,
+    // even when these site files don't exist for your Hadoop installation.
+    Configuration.addDefaultResource("mapred-default.xml");
+    Configuration.addDefaultResource("mapred-site.xml");
+    Configuration.addDefaultResource("yarn-default.xml");
+    Configuration.addDefaultResource("yarn-site.xml");
```

Comment:
Since hadoop 1 does not have {code}yarn-site.xml{code} and {code}yarn-default.xml{code}. Does adding these extra resources affect something in case of hadoop 1?